content/docs/releases/v1.10.0.md

```diff
@@ -0,0 +1,200 @@
+Generated commands could be re-used in performance testing to keep performance testing up-to-date.
+
+[See more details](https://github.com/networkservicemesh/gotestmd/issues/39)
+
+
+### NSC - Add support for K8S PSS restricted/baseline profiles (for hostPath volumes)
+
+K8S 1.25 release deprecates PSP (Pod security policy) and enables PSS (Pod Security Standards)/PSA (Pod Security Admission controller) by default. The PSS has 3 profiles - Privileged, Baseline and Restricted. 
+NSC requires hostPath volumes for unix sockets. HostPath volumes are not permitted in Baseline/Restricted profiles. 
+
+Need a solution for NSC to work in Baseline/Restricted profiles.
+
+Couple of options:
+- NSM CSI driver : Add a NSM CSI driver plugin to mount the hostPath volume.
+- Use network sockets instead of unix sockets and eliminate the need for hostPath volume.
+<!--- Try to give a general description (optional if it overlaps with the content from the Google Docs file) -->
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9218)
+
+
+### Use counters instead of histogram for datapath metrics
+
+Counters are more appropriate for NSM data-path metrics instead of histogram.
+
+[See more details](https://github.com/networkservicemesh/sdk/issues/1464)
+
+
+### Replace govpp from nsm vpp appss to nsm/govpp
+
+In NSM it's super important to use latest patches or be able to quickly fix some issues in vpp. By this reason we need to start use govpp from nsm organization that improves control of the vpp components in the project.
+
+[See more details](https://github.com/networkservicemesh/govpp/issues/2)
+
+
+### Add a new example when one nsc connects to two nses with different services
+
+Status: done/resolved/fixed.
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9294)
+
+
+## System stability fixes and improvements
+
+
+
+### cmd-admission-webhook-k8s generates client IDs properly only for Kind: Pod
+
+cmd-admission-webhook-k8s generated client IDs properly **only if client kind is Pod**.
+
+[See more details](https://github.com/networkservicemesh/deployments-k8s/issues/9170)
+
+
+### AWS: AF_XDP support
+
+
+Root issue: https://github.com/networkservicemesh/cmd-forwarder-vpp/issues/859
+
+AF_XDP socket doesn't work on AWS cluster
+Logs:
+```
+Apr  3 13:24:25.406 [INFO] [cmd:vpp] libbpf: Kernel error message: veth: Peer MTU is too large to set XDP
+Apr  3 13:24:25.406 [INFO] [cmd:vpp] vpp[10508]: af_xdp: af_xdp_load_program: bpf_set_link_xdp_fd(eth0) failed: Numerical result out of range
+Apr  3 13:24:26.563 [ERRO] [cmd:/bin/forwarder] [duration:18.015838ms] [hostIfName:eth0] [vppapi:AfXdpCreate] VPPApiError: System call error #6 (-16)
+panic: error: VPPApiError: System call error #6 (-16)
+```
+
+[See more details](https://github.com/networkservicemesh/integration-k8s-aws/issues/356)
+
+
+### Kubernetes installation
```

Comment:
Might be worth clarifying: "Equinix Metal kubernetes installation"

Reply:
Fixed